cloud/datasync/src/SyncServer.ts

```diff
@@ -0,0 +1,68 @@
+import SyncApi from './SyncApi';
+import SyncOptions, { SyncDataLayerOptions } from './options/SyncGlobalOptions';
+import SyncDataSetOptions from './options/SyncDatasetOptions';
+
+import { setupGlobalHandlers, setupHandlers } from './crud-handlers/HandlerMapper'
+import DataSetHandler from './crud-handlers/DataSetHandler'
+
+import { api } from 'fh-sync'
+
+/**
+ * Implementation for sync server side api
+ */
+export class SyncServer implements SyncApi {
```

Comment:
This class seems to be mostly calling into fh-sync methods, which are singletons from the way `require/import` works, so wouldn't you have some issues with multiple instances of it?

You don't need to have a constructor function/class to implement an API, specially if you don't have instance members and so on. Just an object is okay.